### DISCUSSION TITLE
I have been approved but I do not have the benefits package yet:

### DISCUSSION THREAD

**Discussion by AnYanYi** [community/community#133168] [GitHub Education] [answered]
### Hi everyone,

I have been approved but I do not have the benefits package yet，and It's been 72 hours。
![屏幕截图 2024-07-23 101548](https://github.com/user-attachments/assets/1e412c34-d773-44cf-a53c-dc8e47ab6f40)

**Comment by davevad93** [comment, marked as answer]
Hi @AnYanYi ,

It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AnYanYi  was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470

**Reply by TmkShink** [reply]
Is there any possibility that I have been approved but I cant get the benefits?

**Reply by davevad93** [reply]
Hi @Skylarpopgirl ,

Normally when the benefits are available it means you can access them. Take a look at your [application page](https://education.github.com/discount_requests/application), if it says `Congrats! Your request for @Skylarpopgirl was approved and your benefits will be available soon.` it means you have to wait, When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Skylarpopgirl was approved and your benefits are now available.`

**Reply by TmkShink** [reply]
So I just need to wait and I will never be rejected again right?Thank you for your reply

**Reply by davevad93** [reply]
You're welcome! That's right @Skylarpopgirl and don't worry, if your application has the following label  you won't be rejected because you're already approved, now you have just to wait for GitHub Education to process your request to give you access to your benefits. Be patient because it can take from a few days to a couple of weeks.

![image](https://github.com/user-attachments/assets/adbf5c40-38db-460c-aefe-069dbf107599)

**Reply by snehilshah** [reply]
Hi @davevad93,
I'm in the process of renewal (The application is approved), but I have lost access to the benefits as of yesterday, is that the normal procedure?

**Reply by davevad93** [reply]
Hey @srshah27 , I'm not entirely sure but I think it's the normal procedure because basically they need to reverify your academic status.
